docs/generated/sparse.COO.rst

```diff
@@ -77,6 +77,7 @@ COO
       COO.resize
       COO.transpose
       COO.swapaxes
+      COO.moveaxis
```

Comment:
This will fail to build docs, you should move it to the main module.

Reply:
yes, thanks for the reminder!